keras_retinanet/models/resnet.py

```diff
@@ -77,7 +77,10 @@ def resnet_retinanet(num_classes, backbone='resnet50', inputs=None, modifier=Non
         resnet = modifier(resnet)
 
     # create the full model
-    model = retinanet.retinanet_bbox(inputs=inputs, num_classes=num_classes, backbone=resnet, **kwargs)
+    layer_names = ["res3d_relu", "res4f_relu", "res5c_relu"]
+    layer_outputs = [resnet.get_layer(name).output for name in layer_names]
+    model = retinanet.retinanet_bbox(inputs=inputs, num_classes=num_classes,
+                                     backbone_layers=layer_outputs, **kwargs)
```

Comment:
Unless I'm mistaken the backbone layers can simply be `resnet.outputs[1:]` which would free us from worrying about their names.

Also, I seems a bit excessive to split the call to `retinanet_bbox()` over multiple lines (same for the mobilenet model). 1920x1080 is pretty much the standard resolution these days.